vendor/SimpleDBI/SimpleDBIStatement (copy).php

```diff
@@ -0,0 +1,603 @@
+
+
+
+<!DOCTYPE html>
+<html lang="en" class="">
+  <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# object: http://ogp.me/ns/object# article: http://ogp.me/ns/article# profile: http://ogp.me/ns/profile#">
```

Comment:
Why there is a page here? what is this `vendor/SimpleDBI/SimpleDBIStatement (copy).php` po?